layouts/basic/modules/Users/ListViewContents.tpl

```diff
@@ -101,12 +99,15 @@
 				<tr class="listViewEntries" data-id='{$LISTVIEW_ENTRY->getId()}'
 					data-recordUrl='{$LISTVIEW_ENTRY->getDetailViewUrl()}'
 					id="{$MODULE}_listView_row_{$smarty.foreach.listview.index+1}">
-					<td width="2%" class="{$WIDTHTYPE}">
+					<td class="noWrap leftRecordActions listButtons {$WIDTHTYPE}" >
 						<input type="hidden" name="deleteActionUrl" value="{$LISTVIEW_ENTRY->getDeleteUrl()}">
-						{if $LISTVIEW_ENTRY->isEditable()}
-							<input type="checkbox" value="{$LISTVIEW_ENTRY->getId()}"
-								   title="{\App\Language::translate('LBL_SELECT_SINGLE_ROW')}"
-								   class="listViewEntriesCheckBox"/>
+						{include file=\App\Layout::getTemplatePath('ListViewLeftSide.tpl', $MODULE)}
+						{if $LISTVIEW_ENTRY->get('status') neq 'Active'}
+							<a class="btn btn-sm btn-light" onclick="Settings_Users_List_Js.restoreUser({$LISTVIEW_ENTRY->getId()}, event);">
+								<span class="fas fa-sync-alt"
+									 title="{\App\Language::translate('LBL_RESTORE', $MODULE)}"></span>
+								<span class="sr-only">{\App\Language::translate('LBL_RESTORE', $MODULE)}</span>
+							</a>
```

Comment:
move to record file
see: https://github.com/YetiForceCompany/YetiForceCRM/blob/ab0807f6006f30ab47fcb9df70d37856cc16fbb7/modules/Vtiger/models/Record.php#L1631